modules/ipi-install-network-requirements.adoc

```diff
@@ -36,58 +36,39 @@ For example:
 test-cluster.example.com
 ----
 
-ifeval::[{product-version}>4.7]
-{product-title} 4.8 and later releases include functionality that uses cluster membership information to generate A/AAAA records. This resolves the node names to their IP addresses. Once the nodes are registered with the API, the cluster can disperse node information without using CoreDNS-mDNS. This eliminates the network traffic associated with multicast DNS.
-endif::[]
+{product-title} includes functionality that uses cluster membership information to generate A/AAAA records. This resolves the node names to their IP addresses. Once the nodes are registered with the API, the cluster can disperse node information without using CoreDNS-mDNS. This eliminates the network traffic associated with multicast DNS.
 
-ifdef::upstream[]
-For assistance in configuring the DNS server, check xref:ipi-install-upstream-appendix[Appendix] section for:
-
-- xref:creating-dns-records-on-a-dns-server-option1_{context}[Creating DNS Records with Bind (Option 1)]
-- xref:creating-dns-records-using-dnsmasq-option2_{context}[Creating DNS Records with dnsmasq (Option 2)]
-
-endif::[]
-
-.Dynamic Host Configuration Protocol (DHCP) requirements
+== Dynamic Host Configuration Protocol (DHCP) requirements
 
 By default, installer-provisioned installation deploys `ironic-dnsmasq` with DHCP enabled for the `provisioning` network. No other DHCP servers should be running on the `provisioning` network when the `provisioningNetwork` configuration setting is set to `managed`, which is the default value. If you have a DHCP server running on the `provisioning` network, you must set the `provisioningNetwork` configuration setting to `unmanaged` in the `install-config.yaml` file.
 
 Network administrators must reserve IP addresses for each node in the {product-title} cluster for the `baremetal` network on an external DHCP server.
 
-.Reserving IP addresses for nodes with the DHCP server
+== Reserving IP addresses for nodes with the DHCP server
 
-For the `baremetal` network, a network administrator must reserve a number of IP addresses, including:
+For the `baremetal` network, a network administrator must reserve a number of IP addresses with infinite leases, including:
```

Comment:
This is not true, and I think the bug that prompted this is invalid. I'll leave a comment on the bz as well so we can sort this out.

Reply:
I fixed that comment, but added an additional admonishment about changing IP addresses after installation. Do these changes make sense to you? I want to get get the feedback done for one PR because I need to create separate PRs for 4.8 and 4.9.